presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -18,12 +18,32 @@ To configure the Iceberg connector, create a catalog properties file
 ``etc/catalog/iceberg.properties`` with the following contents,
 replacing the properties as appropriate:
 
+Hive Metastore catalog
+^^^^^^^^^^^^^^^^^^^^^^
+
+Iceberg connector supports the same configuration for
+`HMS <https://prestodb.io/docs/current/connector/hive.html#metastore-configuration-properties>`_
+as Hive connector.
+
```

Comment:
We already have a section to discuss the catalog types supported by the iceberg connector. Would you like to move this to that section (https://prestodb.io/docs/current/connector/iceberg.html#iceberg-catalog-type)?

Reply:
Hi @ChunxuTang, I was thinking wouldn't it be better to have 2 sections for catalog config just like https://prestodb.io/docs/current/connector/iceberg.html#nessie-catalog ?

Reply:
That's a nice idea! Sure. Feel free to put the catalog configs together with separate sections.